src/proxy-page/steps/fixEmbeddedFile.ts

```diff
@@ -0,0 +1,36 @@
+import * as cheerio from 'cheerio';
+import { Step } from '../proxy-page.step';
+import { ContextService } from '../../context/context.service';
+
+/* eslint-disable no-useless-escape, prefer-regex-literals */
+export default (): Step => async (context: ContextService): Promise<void> => {
```

Comment:
I think we can remove async in this function definition, because we don't need to wait for anything.